scripts/build/Jenkins/Jenkinsfile

```diff
@@ -577,13 +577,19 @@ finally {
             )
         }
         node('controller') {
-            step([
-                $class: 'Mailer',
-                notifyEveryUnstableBuild: true,
-                recipients: emailextrecipients([
+            if("${currentBuild.currentResult}" == "SUCCESS") {
+                emailBody = "${BUILD_URL}\nSuccess!"
+            } else {
+                buildFailure = tm('${BUILD_FAILURE_ANALYZER}')
```

Comment:
should you check that the build failure analyzer is on? consider this may be running in an environment that doesn't have it

Reply:
Good point! Will run some tests to see if we can check whether build failure analyzer is on.

Reply:
Tested tm('${BUILD_FAILURE_ANALYZER}') won't fail the build on a Jenkins instance where build failure analyzer is not installed, the failed build log link is included in the email after a failed build.